### DISCUSSION TITLE
Delete single history entry

### DISCUSSION THREAD

**Discussion by cmvanb** [qutebrowser/qutebrowser#7495] [Q&A] [answered]
Is it possible to remove a specific history entry while in command mode? If yes, how?

Apologies if this has been asked before, I wasn't able to find an answer with github or google search.

**Comment by The-Compiler** [comment, marked as answer]
Yup: Hit `Ctrl-D` (bound to `:completion-item-del`).

**Comment by cmvanb** [comment]
Thank you!
